Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -182,6 +182,11 @@ theorem constFormalMultilinearSeries_radius {v : F} :
     (by simp [constFormalMultilinearSeries])
 #align formal_multilinear_series.const_formal_multilinear_series_radius FormalMultilinearSeries.constFormalMultilinearSeries_radius
 
+/-- `0` has infinite radius of convergence -/
```

Comment:
Does this generalise to any constant formal series?

Reply:
Not really, as this is the 0 of that type, not an injection from the field 0.